vmm/src/coredump.rs

```diff
@@ -3,6 +3,10 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#[cfg(target_arch = "x86_64")]
```

Comment:
This is a comment about the commit message, you should remove `Co-authored-by: Sebastien Boeuf <sebastien.boeuf@intel.com>` as I don't think I did anything on this commit.

Reply:
Okay.
The reason is maybe there are some code are removed in some rebase work but I kept the commit message.